### ISSUE TITLE
4chan captcha crashes app on Samsung DeX

### ISSUE THREAD

**Issue by tidux** [K1rakishou/Kuroba-Experimental#655]
How to repeat:

1. Connect Samsung device with DeX mode to external keyboard, mouse, and monitor
2. Open app in DeX mode
3. Type a reply to a thread
4. Submit captcha correctly
5. Click "verify"
6. App crashes, post not sent

**Comment by K1rakishou** [comment]
Yeah this is a Jetpack Compose bug. Can't fix it myself so will have to wait till they fix it.

**Comment by tidux** [comment]
Bummer.  At least using a regular browser works fine in DeX mode.
